retrofit-converters/jaxb/src/main/java/retrofit2/converter/jaxb/ConvertingMediaType.java

```diff
@@ -0,0 +1,6 @@
+package retrofit2.converter.jaxb;
+
+/** Request/response media type to convert by JAXB */
+enum ConvertingMediaType {
+  XML, JSON
```

Comment:
If it's just an API then I don't see why we should restrict the formats to these two? For the [kotlinx.serialization converter][1] you can supply a custom `MediaType` since it's entirely format-agnostic.

 [1]: https://github.com/JakeWharton/retrofit2-kotlinx-serialization-converter/#usage

Reply:
You're right. I kept in mind only an example with EclipseLink MOXy, xml & json.